### PR TITLE
TELCODOCS#1255: Removed all instances of single-node

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s): 4.15

Issue: [TELCODOCS-1255](https://issues.redhat.com/browse/TELCODOCS-1255)

Link to docs preview:
https://69310--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by jakobmoellerdev — COMMENTED** [review]
In general we should probably remove the occurence of OpenShift by a lot since we are already in the OpenShift Docs. For example instead of 
`Creating volume clones in OpenShift`
one can write
`Creating volume clones` since it is clear we are in OpenShift. There were also some minor corrections of `a SNO cluster` vs `an OpenShift` cluster.

Other than that LGTM

**Comment by sr1kar99** [comment]
> In general we should probably remove the occurence of OpenShift by a lot since we are already in the OpenShift Docs. For example instead of `Creating volume clones in OpenShift` one can write `Creating volume clones` since it is clear we are in OpenShift. There were also some minor corrections of `a SNO cluster` vs `an OpenShift` cluster.
> 
> Other than that LGTM

**Comment by sr1kar99** [comment]
@jakobmoellerdev 
Addressed the review comments.
I request you to review the changes.
Thanks.

File: modules/deploying-lvms-on-sno-cluster.adoc [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="lvms-preface-sno-ran_{context}"]
```
[the remark this answers is not in the document]
Comment (sr1kar99) [reply modules/deploying-lvms-on-sno-cluster.adoc:6]: These instances in the Ids and filenames can be ignored as they are not visible in the documentation.

File: modules/deploying-lvms-on-sno-cluster.adoc [hunk 2]
```diff
@@ -4,9 +4,9 @@
...
-= Deploying {lvms} on {sno} clusters
+= Deploying {lvms} on the clusters
```
[the remark this answers is not in the document]
Comment (sr1kar99) [reply modules/deploying-lvms-on-sno-cluster.adoc]: These changes have been incorporated.
Thanks.

File: modules/deploying-lvms-on-sno-cluster.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (jakobmoellerdev) [reply modules/deploying-lvms-on-sno-cluster.adoc:6]: I suggest that we then to a project wide search and replace for the id if possible. If not, feel free to leave as is

[the remark this answers is not in the document]
Comment (sr1kar99) [reply modules/deploying-lvms-on-sno-cluster.adoc:6]: Sure. Will leave as is for now. 
I'll take this up as a separate task.

**Comment by sr1kar99** [comment]
@radeore 
Can you please review this PR?
Thanks.

File: modules/deploying-lvms-on-sno-cluster.adoc [hunk 3]
```diff
@@ -15,30 +15,30 @@ You might need to manually perform such recovery.
...
+Before you deploy {lvms}, be aware of the following limitations:
 
 * You can only create a single instance of the `LVMCluster` custom resource (CR) on an {product-title} cluster.
 * When a device becomes part of the `LVMCluster` CR, it cannot be removed.
 
 [id="lvms-deployment-limitations-for-sno-ran_{context}"]
 == Limitations
 
-For deploying {sno}, LVM Storage has the following limitations:
+For deploying OpenShift, LVM Storage has the following limitations:
```
Comment (skopacz1) [inline modules/deploying-lvms-on-sno-cluster.adoc:41]: Suggestion, feel free to disregard:
```suggestion
For deploying {product-title}, LVM Storage has the following limitations:
```

File: modules/lvms-upgrading-lvms-on-sno.adoc
Comment (skopacz1) [inline modules/lvms-upgrading-lvms-on-sno.adoc:1]: This is somewhat out of scope for your PR, but would you mind replacing Upgrade and Upgrading on the lines you modified in this file to Update and Updating? Update is the preferred term per our [term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update).

**Review by skopacz1 — COMMENTED** [review]
One suggestion and one nit, otherwise looks good to me!

File: modules/lvms-upgrading-lvms-on-sno.adoc
Comment (sr1kar99) [reply modules/lvms-upgrading-lvms-on-sno.adoc:1]: @skopacz1 
Thank you for the review comments.
There were a few more instances of upgrade/upgrading which I have changed to "update/updating".
PTAL.
Thank you!

File: modules/deploying-lvms-on-sno-cluster.adoc (hunk 3 above)
Comment (sr1kar99) [reply modules/deploying-lvms-on-sno-cluster.adoc]: Done!

File: modules/lvms-creating-volume-snapshots-in-single-node-openshift.adoc [hunk 4]
```diff
@@ -16,7 +16,7 @@ You can create volume snapshots based on the available capacity of the thin pool
 
 .Procedure
 
-. Log in to the {sno} for which you need to run the `oc` command.
+. Log in to OpenShift by running the `oc` command.
```
Comment (jeana-redhat) [inline modules/lvms-creating-volume-snapshots-in-single-node-openshift.adoc:19]:
```suggestion
. Log in to {product-title} by running the `oc` command.
```

File: modules/lvms-upgrading-lvms-on-sno.adoc [hunk 5]
```diff
@@ -4,13 +4,13 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="lvms-upgrading-lvms-on-sno_{context}"]
-= Upgrading {lvms} on {sno} clusters
+= Updating {lvms} on clusters
 
-You can upgrade the {lvms-first} Operator to ensure compatibility with your {sno} version.
+You can update the {lvms-first} Operator to ensure compatibility with your OpenShift version.
```
Comment (jeana-redhat) [inline modules/lvms-upgrading-lvms-on-sno.adoc:9]:
```suggestion
You can update the {lvms-first} Operator to ensure compatibility with your {product-title} version.
```

File: storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc [hunk 6]
```diff
@@ -114,9 +114,9 @@ include::modules/lvms-scaling-storage-expand-pvc.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-single-node-openshift-cluster_logical-volume-manager-storage[Scaling up storage by adding capacity to your {sno} cluster]
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-single-node-openshift-cluster_logical-volume-manager-storage[Scaling up storage by adding capacity to your OpenShift cluster]
```
Comment (jeana-redhat) [inline storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc:117]: The linked section doesn't have the product name, remove to match (or update both to `{product-title}`)
```suggestion
* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-single-node-openshift-cluster_logical-volume-manager-storage[Scaling up storage by adding capacity to your cluster]
```

File: storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc [hunk 7]
```diff
@@ -114,9 +114,9 @@ include::modules/lvms-scaling-storage-expand-pvc.adoc[leveloffset=+2]
...
 
-* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-single-node-openshift-cluster-using-rhacm_logical-volume-manager-storage[Scaling up storage by adding capacity to your single-node OpenShift cluster using RHACM]
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-single-node-openshift-cluster-using-rhacm_logical-volume-manager-storage[Scaling up storage by adding capacity to your OpenShift cluster using RHACM]
```
Comment (jeana-redhat) [inline storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc:119]: The linked section doesn't have the product name, remove to match (or update both to `{product-title}`). RHACM = `{rh-rhacm}`.
```suggestion
* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-single-node-openshift-cluster-using-rhacm_logical-volume-manager-storage[Scaling up storage by adding capacity to your cluster using {rh-rhacm}]
```

**Review by jeana-redhat — COMMENTED** [review]
Just a few product name things to address before merge, otherwise LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by sr1kar99** [comment]
@jeana-redhat 
Thank you for the review. Updated the content based on the comment. PTAL.

**Comment by jeana-redhat** [comment]
> @jeana-redhat Thank you for the review. Updated the content based on the comment. PTAL.

LGTM, thanks!

**Pull request merged**